price_feeds_poller.py

```diff
@@ -587,10 +588,10 @@ def avg_fees(pfs):
     return 0
 
 def dry_run_request(bytecode, timeout_secs):
-  cmdline = "npx witnet-toolkit try-query --hex "
+  cmdline = "npx witnet-toolkit trace-query --hex "
   cmdline += bytecode.hex()
-  cmdline += " | tail -n 2 | head -n 1 | awk -F: '{ print $2 }' | sed 's/ //g' | tr -d \"│\""
-  
+  cmdline += " | tail -n 2 | head -n 1 | awk -F: '{ print $3 }' | sed 's/ //g' | tr -d \"│\""
```

Comment:
Why do we need to get the third field in the line instead of the second?

Reply:
Due to the latest changes in the text that the witnet-toolkit prints out to console